float64 p_gain
float64 d_gain
float64 des_motor_ang_A
float64 des_motor_ang_B
